sirepo/template/raydata.py

```diff
@@ -90,6 +90,10 @@ def stateless_compute_catalog_names(data):
     )
 
 
+def stateless_compute_replay_scans(data):
+    return PKDict(data=_request_scan_monitor(PKDict(method="begin_replay", data=data)))
```

Comment:
Make this method name "begin_replay" the same as the stateless compute method "replay_scans" so it is easier to follow the code. I don't have a preference for which name is chosen.

Reply:
f69cb0460a842c98b329b6e5cbe841843fbbb762